test/__snapshots__/optionals.spec.js.snap

```diff
@@ -0,0 +1,364 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
why are there snapshots here?

Reply:
This snapshots are used for test purposes, for check content of files generated by this kickoff, files are auto-generated by jest and updates of this works in same way (check [this link](https://jestjs.io/docs/en/snapshot-testing) for more info). Any other ideas of how test this generator are welcome.